packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1748,6 +1751,9 @@ class StackViewport extends Viewport implements IStackViewport {
 
     this.setCameraNoEvent({ viewUp, viewPlaneNormal });
 
+    // Setting this makes the following comment about resetCameraNoEvent not modifying viewUp true.
+    this.initialViewUp = viewUp;
+
     // Reset the camera to point to the new slice location, reset camera doesn't
```

Comment:
Prior to my changes, this comment was not true. viewUp was getting changed via a call to camera.roll.